r-package/grf/R/average_treatment_effect.R

```diff
@@ -66,61 +122,138 @@
 average_treatment_effect <- function(forest,
                                      target.sample = c("all", "treated", "control", "overlap"),
                                      method = c("AIPW", "TMLE"),
-                                     subset = NULL) {
+                                     subset = NULL,
+                                     debiasing.weights = NULL,
+                                     compliance.score = NULL,
+                                     num.trees.for.weights = 500) {
+
   target.sample <- match.arg(target.sample)
   method <- match.arg(method)
   cluster.se <- length(forest$clusters) > 0
 
-  if (!("causal_forest" %in% class(forest))) {
-    stop("Average effect estimation only implemented for causal_forest")
+  if (method == "TMLE") {
+    loaded <- requireNamespace("sandwich", quietly = TRUE)
+    if (!loaded) {
+      stop("To use TMLE, please install the package `sandwich`.")
```

Comment:
I don't think this message is needed anymore, `sandwich` is in the GRF package requirements (BLP does not check for this)